### PR TITLE
Change buefy.dialogs to csc-ui modals

### PR DESCRIPTION
**Pull request by emm1R**
### Description

Replace buefy-dialogs with csc-ui modals.

### Related issues

GL 944

### CODE REVIEW

**Comment by emm1R** [comment]
Delete works, but if someone could clarify the use cases of this.$route.params.owner, this.$route.params.project, and this.ownerProject (in ObjectTable)?

**Comment by hannyle** [comment]
> Delete works, but if someone could clarify the use cases of this.$route.params.owner and this.$route.params.project?

`this.$route.params.owner` refers to the project owner that shares a Folder/Container to us.

**Comment by hannyle** [comment]
Could we have the DeleteModal looked a bit more similar to [Figma design](https://www.figma.com/proto/N0Y3N0nfyMS2wG3gjwZ201/SD-Connect-Prototype?node-id=2965%3A10952&scaling=min-zoom&page-id=517%3A2532&starting-point-node-id=2965%3A9293&show-proto-sidebar=1) ?

Screenshot 2023-03-10 at 9 45 10

File: swift_browser_ui_frontend/src/components/Containers.vue [hunk 1]
```diff
@@ -56,6 +57,7 @@ export default {
       abortController: null,
       containers: [],
       renderingContainers: [],
+      tableKey: 1,
```
Comment (csc-felipe) [inline swift_browser_ui_frontend/src/components/Containers.vue:60]: What does this do?

Comment (emm1R) [reply swift_browser_ui_frontend/src/components/Containers.vue:60]: I noticed that e.g. if there is only one container at the last page, that the table does not update if it is deleted.

Comment (emm1R) [reply swift_browser_ui_frontend/src/components/Containers.vue:60]: Meaning the page does not update

Comment (csc-felipe) [reply swift_browser_ui_frontend/src/components/Containers.vue:60]: So if you change that value, the table updates?

Comment (csc-felipe) [reply swift_browser_ui_frontend/src/components/Containers.vue:60]: Sounds like a bug in the data-table, no?

**Comment by csc-felipe** [comment]
Can be rebase to fix tests, now that #1025 was merged.

File: swift_browser_ui_frontend/src/components/Containers.vue (hunk 1 above)
Comment (emm1R) [reply swift_browser_ui_frontend/src/components/Containers.vue:60]: Yes, if you change the page, the table updates.

Comment (emm1R) [reply swift_browser_ui_frontend/src/components/Containers.vue:60]: Hmm, maybe I have an old version of csc-ui

Comment (emm1R) [reply swift_browser_ui_frontend/src/components/Containers.vue:60]: No, update didn't work.

Comment (emm1R) [reply swift_browser_ui_frontend/src/components/Containers.vue:60]: I'll remove the key and add an issue to csc-ui

File: swift_browser_ui_frontend/src/components/DeleteModal.vue [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+      {{ message }}
+
+      <c-card-actions justify="end">
+        <c-button 
+          outlined
+          @click="toggleDeleteModal"
+          @keyup.enter="toggleDeleteModal"
+        >
+          Cancel
+        </c-button>
```
Comment (hannyle) [inline swift_browser_ui_frontend/src/components/DeleteModal.vue:17]: We have `Cancel` term translated in lang.js. Could we also add it here?

File: swift_browser_ui_frontend/src/common/globalFunctions.js [hunk 3]
```diff
@@ -31,6 +31,16 @@ export function toggleCopyFolderModal(folderName, sourceProjectId) {
...
 
+export function toggleDeleteModal(objects, containerName) {
+  store.commit("toggleDeleteModal", true);
+  if (objects) {
+    store.commit("setDeletableObjects", objects);
+  }
+  if (containerName) {
+    store.commit("setFolderName", containerName);
+  }
+}
```
Comment (hannyle) [inline swift_browser_ui_frontend/src/common/globalFunctions.js:42]: I think we could also add `modifyBrowserPageStyles()` for functions that handle opening/closing DeleteModal. 
This one helps prevent scrolling in the background when a modal is opened :)

Comment (csc-felipe) [reply swift_browser_ui_frontend/src/common/globalFunctions.js:42]: I've been wondering what was it for

File: swift_browser_ui_frontend/src/components/DeleteModal.vue (hunk 2 above)
Comment (emm1R) [reply swift_browser_ui_frontend/src/components/DeleteModal.vue:17]: ok

File: swift_browser_ui_frontend/src/common/globalFunctions.js (hunk 3 above)
Comment (hannyle) [reply swift_browser_ui_frontend/src/common/globalFunctions.js]: I see. We could fix this jumping to the top of the page in another PR :)

**Review by hannyle — APPROVED** [review]

File: swift_browser_ui_frontend/src/common/globalFunctions.js (hunk 3 above)
Comment (hannyle) [reply swift_browser_ui_frontend/src/common/globalFunctions.js]: This PR fixes the jumping issue: https://github.com/CSCfi/swift-browser-ui/pull/1024

**Pull request merged**